test/reaping-faulty-nodes.js

```diff
@@ -217,10 +217,21 @@ test2('tombstone should be gossiped with flag when applied as state', getCluster
             // clear all pings that happened before we gossiped the tombstone
             dsl.consumePings(t, tc),
 
-            // Wait for a ping from the SUT and validate that it does not gossip about the tombstone
-            dsl.validateEventBody(t, tc, {
-                type: events.Types.Ping,
-                direction: 'request'
+            // Wait for a ping from the SUT and validate that it does not gossip about the tombstone.
```

Comment:
Even though this line came from the existing code, when I read it again here it seemed a bit misleading. Because we are making sure it _does_ gossip about the tombstone, but we make sure it does not gossip the tombstone state. If you can add `state` or `as a state` to the end of the line it does make more sense.